applications/picocdn-node-vm/README.md

```diff
@@ -0,0 +1,87 @@
+
+
+## **Pre Requisites – Resources Required**
+
+| **picoCDN-VM Variants**       | **(S) Small**        | **(M) Medium**       | **(L) Large**        |
+| ----------------------------- | -------------------- | -------------------- | -------------------- |
+| **Service Performance**       |                      |                      |                      |
+| Content Delivery              | 1  Gbps              | 5  Gbps              | 10  Gbps             |
+| **Required Resources**        | -------------------- | -------------------- | -------------------- |
+| Compute* (vCores)             | 4-8                  | 12+                  | 24+                  |
+| Memory (RAM)                  | 16 GB                | 64 GB                | 128 GB               |
+| Storage (System+Logs minimum) | 100  GB              | 200  GB              | 500  GB              |
+| Storage (Content recommended) | 2-5 TB SSD           | 10-15 TB SSD         | 15-30 TB SSD         |
+| Storage Performance           | 1.5 GBps  800 IOPS   | 6 GBps   1600 IOPS   | 12 GBps    3200 IOPS |
+| Host OS                       <td colspan=3> picoCDN7 Custom Image </td></tr>
+| Network                       <td colspan=3> 1G / 10G / 25G+ via  Virtio / Direct-Attach / SR-IOV</td></tr>
+
+
+## **Where to Purchase**
+
+The picoCDN custom image can be obtained from the picoNETS support portal. For more information visit [www.piconets.com](https://www.piconets.com) or contact us via email on info@piconets.com
+
+
+## Pre Requisites - Installing OpenNESS
+
+|**Configuration**                   |                       |
+|  ----------------------------------|-----------------------|
+| OpenNESS Version                   | 20.06                 |
+| Flavor Used 			     | kubevirt VM           |
+| Distribution	                     | IDO       	     |
+
+This document assumes that OpenNESS was installed through [OpenNESS playbooks](https://github.com/open-ness/specs/blob/master/doc/getting-started/network-edge/controller-edge-node-setup.md). Certain configurations had been changed to support VM deployment on OpenNESS for network edge following [openness-network-edge-vm-support.md](https://github.com/open-ness/specs/blob/master/doc/applications-onboard/openness-network-edge-vm-support.md).

```

Comment:
The supported OpenNESS version is 20.06, could you update the links to the OpenNESS documentation to point to the 20.06 version please?

Reply:
Can you ensure this is working with latest OpenNESS release 21.03. The OpenNESS 20.06 release is old and may not be supported.

Reply:
Please update the Readme.md with the troubleshooting steps to use the 20.06 OpenNESS, if they have to use 20.06.

Reply:
We are currently in the process of getting access to a test deployment of 21.03.x at the network builders lab, where we will be validating functionality, especially with the updated Calico CNI. We intend to update and push the modified charts once that testing is complete.

Reply:
Resolving via the following edits :
1. Updated the links to specifically reference the 20.06 tagged versions.

Reply:
Resolving this Suggestion via the following edits 
1. Updated the relevant links to point to the 20.06 tagged versions of the docs on the openness github repo.
2. Additional clarification added to specifically mention that version 20.06 is the required version.